types/stylelint/index.d.ts

```diff
@@ -577,6 +594,7 @@ declare namespace stylelint {
 	 * A lint problem.
 	 */
 	export type Problem = {
+		fix?: () => void | never;
```

Comment:
[suggest] Can you move `fix?` to the end of this type's definition to clarify that it's added later?